docs/routing-architecture/routing-architecture.md

```diff
@@ -214,7 +214,7 @@ class PackageShow extends Component {
 
   render() {
     // get the model off props that was passed into the View
-    let { model, fetchPackageTitles, intl } = this.props;
+    let { model, fetchPackageTitles } = this.props;
```

Comment:
Jeffrey, I would suggest to write all the destructured props on the new line. I understand it is arguable
The big advantede of it is in tracking of fuether changes e.g. if changed one prop than git will track only that prop line change.
I belive it will decrease the amout of merge conflicts and makes I belive more readable code.